packages/code-du-travail-data/dump.js

```diff
@@ -43,7 +44,9 @@ const dump = async () => {
   } else {
     console.error(`no nlp`);
   }
-  for await (const docs of cdtnDocumentsGen()) {
+  for await (const docsNoIds of cdtnDocumentsGen()) {
+    // add CDTN specific ids to docs
+    const docs = docsNoIds.map(hashId);
```

Comment:
I wonder if we could pass directyl source + id to the function and retrun the cdtnId. so the map function will have to call it and deal with the result 
ex 
```js
docsNoIds.map((obj) => {
   cdtnId = hashId(obj.source + obj.id)
   return {...obj, cdtnId}
})
});